src/routers/secure/items.py

```diff
@@ -308,15 +308,18 @@ class RetryResponse(BaseModel):
 async def retry_items(request: Request, ids: str) -> RetryResponse:
     """Re-add items to the queue"""
     ids = handle_ids(ids)
-    try:
-        media_items_generator = get_media_items_by_ids(ids)
-        for media_item in media_items_generator:
-            request.app.program.em.cancel_job(media_item._id)
-            await asyncio.sleep(0.1)  # Ensure cancellation is processed
-            # request.app.program.em.add_item(media_item)
-            request.app.program.em.add_event(Event("RetryItem", media_item._id))
-    except ValueError as e:
-        raise HTTPException(status_code=400, detail=str(e))
+    for id in ids:
+        try:
+            item = next(get_media_items_by_ids([id]), None)
+            if item:
+                with db.Session() as session:
+                    item.scraped_at = None
+                    item.scraped_times = 0
```

Comment:
do we want to go back to 0 here, or possibly set to 1 ?

Reply:
Both work for me!

Reply:
I would do 1, but both will work fine.